driver-core/src/test/java/com/datastax/driver/core/SessionErrorTest.java

```diff
@@ -0,0 +1,89 @@
+/*
+ *      Copyright (C) 2012-2015 DataStax Inc.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+package com.datastax.driver.core;
+
+import org.apache.log4j.Level;
+import org.jboss.byteman.contrib.bmunit.BMNGListener;
+import org.jboss.byteman.contrib.bmunit.BMRule;
+import org.jboss.byteman.contrib.bmunit.BMUnitConfig;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Listeners;
+import org.testng.annotations.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.fail;
+
+/**
+ * Simple test of the Sessions methods against a one node cluster.
+ */
+@BMUnitConfig(loadDirectory = "target/test-classes")
+@Listeners(BMNGListener.class)
+@CCMConfig(createCluster = false)
+public class SessionErrorTest extends CCMTestsSupport {
+
+    private Cluster cluster1;
+    private Cluster cluster2;
+
+    @BeforeMethod
+    public void setUp() throws Exception {
+        cluster1 = register(createClusterBuilder()
+                .addContactPointsWithPorts(ccm().addressOfNode(1)).build()).init();
+        cluster2 = register(createClusterBuilder()
+                .addContactPointsWithPorts(ccm().addressOfNode(1)).build()).init();
+    }
+
+    @Test
```

Comment:
Do we intend for these to run as part of our test harness?  Since the Test annotation lacks a group definition they aren't running.